server/src/controller.py

```diff
@@ -0,0 +1,30 @@
+from config import config
+from sanic.response import json
+import rest_service
+
+def get_heartbeat(request):
+    return json(ok('hello world'))
```

Comment:
In the future we can add a bit more information in this heartbeat such as versioning and dependency health (databases...)